jupyter-mime.el

```diff
@@ -329,10 +337,9 @@ aligns on the current line."
   "Parse and insert the HTML string using `shr'."
   (jupyter-with-insertion-bounds
       beg end (insert html)
-    ;; TODO: We can't really do much about javascript so
-    ;; delete those regions instead of trying to parse
-    ;; them. Maybe just re-direct to a browser like with
-    ;; widgets?
+    ;; Maybe open the html in external browser (e.g. if it has javascript)
+    (jupyter-insert-html--maybe-browse-url beg end)
```

Comment:
I think this should be dependent on kernel language. Although, it doesn't do it now, the Plots package from IJulia used to send a lot of embedded scripts to setup javascript plots before plotting and this javascript wasn't meant for viewing in a browser.

We can do something like

```emacs-lisp
(cl-defmethod jupyter-insert ((_mime (eql :text/html)) data
                              &context (jupyter-lang R)
                              &optional _metadata)
  (jupyter-with-insertion-bounds
      beg end (cl-call-next-method)
    (jupyter-insert-html--maybe-browse-url beg end)))

(cl-defmethod jupyter-insert :around ((_mime (eql :text/html)) data
                                      &optional _metadata)
  (jupyter-with-insertion-bounds
      beg end (cl-call-next-method)
    (jupyter--delete-script-tags beg end)))
```

What happens here is that the `:around` method gets called first which will call the `R` method, which in turn calls the base definition already present. So the `R` method gets a chance at checking from script tags before they get removed from the html in the `:around` method.